README.md

```diff
@@ -3,7 +3,7 @@
 <a name="readme-top"></a>
 
 <div align="center">
- <h3><b>Awesome-book is my demo project for learning  js object and arrays</b></h3>
+ <h3><b>Awesome-book is my demo project for learning  js object and arrays , it has new es6 syntax</b></h3>
```

Comment:
- [ ] I am requesting this change here because you have not made changes to the required file. Kindly note this project is a refactor of a previous project and so the look of the pages should still feel the same. Kindly move your footer to the bottom of all pages.

  ![image](https://user-images.githubusercontent.com/36542986/221782672-54838d97-8484-4872-ad11-b334c601bdf3.png)

- [ ] **[OPTIONAL]** When I remove a book the List page appears blank with no books until I navigate away and back to the List page before I see the remaining books. I think it would be a good idea to fix that.